cypress/support/step-definitions/drawer-steps.js

```diff
@@ -0,0 +1,33 @@
+import {
+  drawerToggle,
+  drawerSidebar,
+  drawerSidebarContent,
+} from '../../locators/drawer';
+
+When('I click on Drawer arrow {int} time(s)', (count) => {
+  for (let i = 0; i < count; i++) {
+    drawerToggle().first().click();
+  }
+});
+
+Then('sidebar should have class {word}', (className) => {
+  drawerSidebar().should('have.class', className);
+});
+
+Then('sidebar text is visible', () => {
+  drawerSidebarContent().find('li:nth-child(1)').should('have.text', 'link a')
```

Comment:
1. Could we move this to index? And after that use it passing the index of element?
```suggestion
  drawerSidebarContentInnerElement(1).should('have.text', 'link a')
```
2. What about verification of `link b` and `link c`?

Reply:
fixed 07d78ca